src/ImageTransformations.jl

```diff
@@ -1,7 +1,7 @@
 __precompile__()
 module ImageTransformations
 
-using CoordinateTransformations, Interpolations, OffsetArrays, StaticArrays, Colors, ColorVectorSpace, ImageCore
+using CoordinateTransformations, Interpolations, OffsetArrays, StaticArrays, FixedPointNumbers, Colors, ColorVectorSpace, ImageCore
```

Comment:
Thanks! Turns out I had a rebase-induced bug, so this has to do another round anyway.